site_scons/site_tools/md2html.py

```diff
@@ -68,75 +81,108 @@ def _getTitle(mdBuffer: io.BytesIO, isKeyCommands: bool = False) -> str:
 		# Make next read at start of buffer
 		mdBuffer.seek(0)
 		# Remove heading hashes and trailing whitespace to get the tab title
-		title = mdBuffer.readline().decode("utf-8").strip().lstrip("# ")
+		title = mdBuffer.readline().strip().lstrip("# ")
 
 	return title
 
 
+def _createAttributeFilter() -> dict[str, set[str]]:
+	# Create attribute filter exceptions for HTML sanitization
+	import nh3

```

Comment:
Should this import go at the top? PEP 8 advises that imports should usually be placed at the top of a file, except in certain cases like circular import issues and optimising performance when using large libraries that are not always used.

Reply:
This can't be done in these scons scripts